### ISSUE TITLE
モデルの学習時のエラー（AIモデルの性能を上げる方法）

### ISSUE THREAD

**Issue by Yogurt-neesan** [karaage0703/karaage-ai-book#41]
**ページ数**

書籍ではなく、GitHubのコードをコピペして実行しています。
「AIモデルの性能を上げる方法」（02_karaage_ai_book_image_classification_performance.ipynb）において、
「データの水増し」の「AIモデルの学習」のところです。

**内容**

上記箇所にて、
「%%time
history = model.fit(train_aug_data, epochs=EPOCHS, validation_data=validation_data, verbose=1)
     」
実行時に
「Node: 'categorical_crossentropy/softmax_cross_entropy_with_logits'
logits and labels must be broadcastable: logits_size=[8,3] labels_size=[8,4]
	 [[{{node categorical_crossentropy/softmax_cross_entropy_with_logits}}]] [Op:__inference_train_function_37264]」
というエラーが出ます。
パラメータが8,3 と8,4で違っているのが原因でしょうか？
コード自体はコピペしているのでタイプミスの可能性はかなり低いと考えられます。
よろしくお願い致します。

**Comment by karaage0703** [comment]
@Yogurt-neesan 
連絡ありがとうございます。以下Notebookで動作確認したところ、ご指摘の箇所以外でエラーが出たため修正いたしました。

[02_karaage_ai_book_image_classification_performance.ipynb](https://github.com/karaage0703/karaage-ai-book/blob/master/ch02/02_karaage_ai_book_image_classification_performance.ipynb)

ノートブックを修正し、学習が正しくできることを確認しました。
![learning_confirmation](https://github.com/karaage0703/karaage-ai-book/assets/5562157/29737f85-f15c-4657-85d3-582724516a3d)

一度ノートブックをそのまま実行して確認いただき、その上でエラーになっている箇所との差分を確認いただけましたら幸いです。

**Comment by karaage0703** [comment]
一旦closeさせていただきます。また問題ありましたらOpenください。

**Comment by Yogurt-neesan** [comment]
すみません。ありがとうございます。
再度実行したらこれより前の部分でエラーが出てしまったので、
別のissueに上げさせていただいています。
どうぞよろしくお願い致します。
